lang/en/website-notice.yaml

```diff
@@ -0,0 +1,12 @@
+banner:
+  title: Bitcoincash.org and its owners no longer accurately represent Bitcoin Cash!
+  sub: Click to find out more
+
+title: Notice to services that use Bitcoin Cash (BCH)
+h1: <span class="text-muted">Bitcoincash.org</span> is unreliable for information regarding the BCH network
+h2: Recommended actions
+p1: Due to the decentralized nature of the BCH network, there is no official reference website for general Bitcoin Cash information. Unfortunately, the most popular of those informational websites, bitcoincash.org, has been publishing information that does not reflect the current reality of the BCH network and the November 15th 2020 scheduled upgrade.
```

Comment:
Odd mix of long and short names. It's also a pretty complicated sentence. I recommend this:

> Due to the decentralized nature of Bitcoin Cash, there is no official website for the network as a whole.